### PR TITLE
Add TIA and fix issues

### PR DESCRIPTION
**Pull request by tobyash86**
Changes done:

- fixed versions
- added TIA content
- fixed links
- fixed lists indentation
- fixed minor detected issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,99 +108,134 @@ To upload reports in other formats, modify your workflow by adding  the `upload-
...
+```yaml
+- name: Run Parasoft dotTEST
+  uses: parasoft/run-dottest-action@2.0.0
+  with:
+    testConfig: 'Run VSTest Tests'
+    referenceReportFile: PATH_TO_REPORT_FILE_ON_AGENT
+    referenceCoverageFile: PATH_TO_COVERAGE_FILE_ON_AGENT
+```
+
+Reference files passed as arguments should be generated by analysis made on the full testing scope (without TIA).
```
Comment (ijarosz) [inline README.md:221]: Please change "analysis made" to "analysis performed"

File: README.md [hunk 2]
```diff
@@ -104,99 +108,134 @@ To upload reports in other formats, modify your workflow by adding  the `upload-
...
+3. Configure the following settings for dotTEST to limit the scope of analysis to files that are different between the current working branch and the reference branch:
+
+    ```yaml
+    scope.scontrol.files.filter.mode=branch
+    scope.scontrol.ref.branch=origin/main
+    ```
+
+    See the [scope.scontrol.files.filter.mode](https://docs.parasoft.com/display/DOTTEST20222/Scope+and+Authorship+Settings#ScopeandAuthorshipSettings-scope.files.time.filter.modescope.scontrol.files.filter.mode) parameter description for details.
+
+#### Executing limited scope of tests with Test Impact Analysis
```
Comment (ijarosz) [inline README.md:208]: Executing limited scope of tests -> Executing a Limited Scope of Tests

File: README.md [hunk 3]
```diff
@@ -104,99 +108,134 @@ To upload reports in other formats, modify your workflow by adding  the `upload-
...
-1. Configure the following settings for dotTEST to limit the scope of analysis to files that are different between the current working branch and the reference branch:
-
- ```yaml
- scope.scontrol.files.filter.mode=branch
- scope.scontrol.ref.branch=origin/main
- ```
- See the [scope.scontrol.files.filter.mode](https://docs.parasoft.com/display/DOTTEST20221/Scope+and+Authorship+Settings#ScopeandAuthorshipSettings-scope.files.time.filter.modescope.scontrol.files.filter.mode) parameter description for details.
+dotTEST provides a way to analyze only the current differences affecting your code compared to a baseline. This feature saves time as only specific fragments of code are analyzed or specific tests are executed to make sure recent changes do not introduce any regressions to your code. Thanks to the limited scope the response is available much sooner compared to the analysis of the entire project.
+
+#### Narrowed down Static Analysis
```
Comment (ijarosz) [inline README.md:183]: I suggest changing the title to "Limiting Static Analysis"

File: README.md [hunk 4]
```diff
@@ -104,99 +108,134 @@ To upload reports in other formats, modify your workflow by adding  the `upload-
...
- - By configuring options in Parasoft dotTEST tool. We recommend creating a `dottestcli.properties` file that includes all the configuration options and adding the file to dotTEST's working directory  - typically, the root directory of your repository. This allows dotTEST to automatically read all the configuration options from that file. See [Parasoft dotTEST User Guide](https://docs.parasoft.com/display/DOTTEST20221) for details.
- 
+
+- By customizing the `Run Parasoft dotTEST` action directly in your GitHub workflow. See [Action Parameters](#action-parameters) for a complete list of available parameters.
+- By configuring options in Parasoft dotTEST tool. We recommend creating a `dottestcli.properties` file that includes all the configuration options and adding the file to dotTEST's working directory  - typically, the root directory of your repository. This allows dotTEST to automatically read all the configuration options from that file. See [Parasoft dotTEST User Guide](https://docs.parasoft.com/display/DOTTEST20222) for details.
```
Comment (ijarosz) [inline README.md:115]: double space before - typically

File: README.md [hunk 5]
```diff
@@ -104,99 +108,134 @@ To upload reports in other formats, modify your workflow by adding  the `upload-
...
-
-1. Configure source control settings. See [Connecting to Source Control](https://docs.parasoft.com/display/DOTTEST20221/Connecting+to+Source+Control) for details.
-1. Configure the following settings for dotTEST to limit the scope of analysis to files that are different between the current working branch and the reference branch:
-
- ```yaml
- scope.scontrol.files.filter.mode=branch
- scope.scontrol.ref.branch=origin/main
- ```
- See the [scope.scontrol.files.filter.mode](https://docs.parasoft.com/display/DOTTEST20221/Scope+and+Authorship+Settings#ScopeandAuthorshipSettings-scope.files.time.filter.modescope.scontrol.files.filter.mode) parameter description for details.
+dotTEST provides a way to analyze only the current differences affecting your code compared to a baseline. This feature saves time as only specific fragments of code are analyzed or specific tests are executed to make sure recent changes do not introduce any regressions to your code. Thanks to the limited scope the response is available much sooner compared to the analysis of the entire project.
```
Comment (ijarosz) [inline README.md:181]: response is available ->test results are available

File: README.md (hunk 4 above)
Comment (tobyash86) [reply README.md:115]: I think it is okay here, double space should be applied when these would be subitems of list items

**Pull request merged**